t/Geometry.t

```diff
@@ -21,6 +21,16 @@ ddg_spice_test(
         '/js/spice/geometry/',
         call_type => 'self',
         caller => 'DDG::Spice::Geometry'
+    ),
+    'area of circle' => test_spice(
+        '/js/spice/geometry/',
+        call_type => 'self',
+        caller => 'DDG::Spice::Geometry'
+    ),
+    'perimeter of square' => test_spice(
+        '/js/spice/geometry/',
+        call_type => 'self',
+        caller => 'DDG::Spice::Geometry'
```

Comment:
Hallo, sorry for not replying sooner. I am a bit busy at the moment, but I am planning to continue working on the tests next Tuesday.